<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtNetwork</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEditServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditClient">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonServerActivateDeactivate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClientConnectDisconnect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelServer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClient">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClientRemoteAddress">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClientRemoteAddress">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClientRemotePort">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1999</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClientRemotePort">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServerPort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1999</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelServerPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClientLocalAddress">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Address = ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClientLocalPort">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>63</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Port = ?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServerMessageToSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClientMessageToSend">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonServerSend">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClientSend">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value = 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
